buildshiprun/handler_test.go

```diff
@@ -175,3 +175,29 @@ func Test_getReadOnlyRootFS_override(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func Test_getConfig(t *testing.T) {
+
+	os.Setenv("scaling_max_limit", "")
```

Comment:
Please use separate tests for each case or a test scenario slice like we do in Derek with a name for each case.